iotdb/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java

```diff
@@ -488,6 +533,22 @@ public TSExecuteStatementResp executeStatement(TSExecuteStatementReq req) throws
     }
   }
 
+  /**
+   * Set consistency level
+   */
+  public boolean execSetConsistencyLevel(String statement) throws Exception {
+    if (statement == null) {
```

Comment:
um.... you never return true...

Reply:
Yes, operation `set consistency level` in stand-alone version IoTDB doesn't support, so it can not return true...

Reply:
Maybe it is more straight-forward to just throw an exception and tell the user that this operation is illegal.

Reply:
Well，`return true` means the statement is `SetConsistencyLevel` sql and `return false` means the statement is other operation, and the method which called `execSetConsistencyLevel()` will continue to do other operation to execute statement, so I cannot throw Excepetion without judge.

Reply:
So the caller does not know what exactly the statement is and still calls this function? Sounds a bit weird. I suggest mark it as a "todo" and we add it into antlr in the future.

Reply:
Well,  `execAdminCommand()` is the same usage as `execSetConsistencyLevel ()`. When execute `executeStatement()` method, firstly check whether it's AdminCommand, return result if it's, else continue; secondly check whether it's SetConsistencyLevel, return result if it's, else continue; thirdly execute Physical Plan.